docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/.metrics-exposer.md

```diff
@@ -0,0 +1,127 @@
+
+### Starting Nebula Graph
+
+Refer to [Get Started](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-CN/1.overview/2.quick-start/1.get-started.md) to start **Nebula Graph** services.
+
+### Starting nebula-prom-transformer
+
+nebula-prom-transformer is a tool that converts **Nebula Graph** metrics into Prometheus interpretable format. It pulls data from **Nebula Graph** and exposes the data endpoints. You can configure Prometheus to pull data from the specified endpoints by referring to the [README](https://github.com/Shylock-Hg/nebula-prom-transformer) documentation.
+
+### Configuring and Start Prometheus
+
+This section introduces how to configure Prometheus and make it pull metrics data from the configured endpoints. For details on Prometheus installation and configuration, please refer to Prometheus [Official Documentation](https://prometheus.io/docs/prometheus/latest/getting_started/). In this section, we only modify the endpoints that pull the metrics data. An example configuration file `prometheus.yml` is as follows.
+
+```yaml
+# my global config
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# Alertmanager configuration
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      # - alertmanager:9093
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+  # - "first_rules.yml"
+  # - "second_rules.yml"
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any time series scraped from this config.
+  - job_name: 'prometheus'
+
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+
+    static_configs:
+    - targets: ['localhost:11001', 'localhost:12001', 'localhost:13001']
+```
+
+As shown above, for default configuration **Nebula Graph**, i.e. the single node, we only need to start three nebula-prom-transformer services to pull **Nebula Graph** metrics data and monitor the 11000, 12000, and 13000 endpoints data. For non-default configuration or clusters, you need to expose all the HTTP endpoints of all services to Prometheus.
+
+### Checking Metrics via Prometheus
+
+After executing the above three steps successfully, all the configurations are completed and **Nebula Graph** and Prometheus are connected. Now you can access the graphical operation interface provided by Prometheus through a browser, input `http: // localhost: 9090` in the browser, then input `add_edges_latency_bucket` in the query box of Prometheus, click the `execute` button to check the corresponding metrics values. Consider the following example:
```

Comment:
Now you can access the graphical operation interface provided by Prometheus through a browser. 
1. Input `http: // localhost: 9090` in the browser.
2. Input `add_edges_latency_bucket` in the query box of Prometheus.
3. Click the `execute` button to check the corresponding metrics values. 

Consider the following example: